SampleBrowser/SampleBrowser/Assets.xcassets/crossCircleFillLarge.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "crossCircleFillLarge.pdf",
```

Comment:
This image has to be added to Client as well as the expectation is that every image added to `StandardImageIdentifiers` exists in Client as well.